### PR TITLE
6 1j login page

### PR DESCRIPTION
**Pull request by larroyo3**
close#6
--  
J'avais pas de maquette donc j'ai fait au mieux mdrrr, je remettrai à jour dès que j'ai une maquette

### CODE REVIEW

File: app/src/main/java/com/bouboulecorp/thurii/LogIn/LogInActivity.kt [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_login)
+    }
+
+    fun goToMainActivity(view: View) {
+        val intent = Intent(this, MainActivity::class.java)
+        startActivity(intent)
+    }
+
+    fun changeStatePassword(view:View) {
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/LogIn/LogInActivity.kt:22]: 🚫 **[ktlint]** <colon-spacing> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing spacing after ":"

File: app/src/main/java/com/bouboulecorp/thurii/LogIn/LogInActivity.kt [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        val editTextPassword = findViewById<EditText>(R.id.editTextTextPassword_login)
+
+        if (editTextPassword.inputType == 129) {       //set "textPassword" to "textVisiblePassword"
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/LogIn/LogInActivity.kt:25]: 🚫 **[ktlint]** <no-multi-spaces> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unnecessary long whitespace

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/LogIn/LogInActivity.kt:25]: 🚫 **[ktlint]** <comment-spacing> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing space after //

File: app/src/main/java/com/bouboulecorp/thurii/LogIn/LogInActivity.kt [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            editTextPassword.inputType = 145
+        }
+        else if (editTextPassword.inputType == 145) {  //set "textVisiblePassword" to "textPassword"
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/LogIn/LogInActivity.kt:28]: 🚫 **[ktlint]** <keyword-spacing> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected newline before "else"

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/LogIn/LogInActivity.kt:28]: 🚫 **[ktlint]** <no-multi-spaces> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unnecessary long whitespace

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/LogIn/LogInActivity.kt:28]: 🚫 **[ktlint]** <comment-spacing> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing space after //

File: app/src/main/java/com/bouboulecorp/thurii/LogIn/LogInActivity.kt [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+            editTextPassword.inputType = 129
+        }
+
+        //placing cursor at the end of the text
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/LogIn/LogInActivity.kt:32]: 🚫 **[ktlint]** <comment-spacing> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing space after //

File: app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationActivity.kt [hunk 5]
```diff
@@ -47,7 +47,7 @@ class RegistrationActivity : AppCompatActivity() {
                 replaceFragment(FragmentRegistrationSport())
                 progress.setProgress(50)
             }
-            if (progress.progress == 25) {
+            else if (progress.progress == 25) {
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationActivity.kt:50]: 🚫 **[ktlint]** <keyword-spacing> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected newline before "else"

File: app/src/main/java/com/bouboulecorp/thurii/LogIn/LogInActivity.kt [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+        val intent = Intent(this, MainActivity::class.java)
+        startActivity(intent)
+    }
+
+    fun changeStatePassword(view: View) {
+        val editTextPassword = findViewById<EditText>(R.id.editTextTextPassword_login)
+
+        if (editTextPassword.inputType == 129) { // set "textPassword" to "textVisiblePassword"
+            editTextPassword.inputType = 145
+        } else if (editTextPassword.inputType == 145) { // set "textVisiblePassword" to "textPassword"
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/LogIn/LogInActivity.kt:27]: 🚫 **[ktlint]** <max-line-length> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Exceeded max line length (100) (cannot be auto-corrected)

**Review by claraxuxu — COMMENTED** [review]
all good

**Review by Neyross — APPROVED** [review]
good

**Pull request merged**